src/hooks/Movie/useFeaturedMovies.js

```diff
@@ -0,0 +1,13 @@
+import useSWR from "swr";
+import API from "api/moviedb.instance";
+
+export default function useFeaturedMovies() {
+    const fetcher = () => API.get("movie/now_playing").then((res) => res.data);
```

Comment:
Move fetcher outside the component as to not get caught up in re renders. Or you can move the fetchers to `utils/api` (optional).

Like in `utils/api`, it will contain your all your APIs, along with the fetchers.

Ex: `utils/api/movie.js`
```javascript
export async function getNowPlayingMovies() {
  const { data } = API.get('movie/now_playing');

  return data;
}
```

Then in this file you can do:
```javascript
import { getNowPlayingMovies } from 'utils/api/movies';

export default function useFeaturedMovies() {
  const { data, error } = useSWR('api/featured', getNowPlayingMovies);
```